network/wsPeer.go

```diff
@@ -168,6 +168,12 @@ const disconnectRequestReceived disconnectReason = "DisconnectRequest"
 const disconnectStaleWrite disconnectReason = "DisconnectStaleWrite"
 const disconnectDuplicateConnection disconnectReason = "DuplicateConnection"
 const disconnectBadIdentityData disconnectReason = "BadIdentityData"
+const disconnectUnexpectedTopicResp disconnectReason = "UnexpectedTopicResp"
+const disconnectExpiredTopicResp disconnectReason = "ExpiredTopicResp"
+
+// If the peer sends us a block response after this threshold
+// we should disconnect from it
+const blockResponseDisconnectThreshold = 60 * time.Second
 
```

Comment:
both `disconnectExpiredTopicResp` and `blockResponseDisconnectThreshold` not used now

Reply:
Thank you removed